src/ng/templateRequest.js

```diff
@@ -86,7 +88,7 @@ function $TemplateRequestProvider() {
           cache: $templateCache,
           transformResponse: transformResponse
         }, httpOptions))
-        ['finally'](function() {
+        .finally(function() {
```

Comment:
I agree, let's not change `[finally]` to `.finally` (same with catch). We can make a decision about "officially" stopping support for Android 2.3 in 1.6